scripts/docker/alpine/Dockerfile

```diff
@@ -10,34 +10,37 @@ RUN apk add --no-cache \
   eudev-dev \
   linux-headers \
   perl \
-  rust
-
-WORKDIR /parity
-COPY . /parity
-RUN cargo build --release --target x86_64-alpine-linux-musl --verbose
-RUN strip target/x86_64-alpine-linux-musl/release/parity
+  rust \
+  git
 
+WORKDIR /openethereum
+COPY . /openethereum
+RUN cargo build --release --features final --target x86_64-alpine-linux-musl --verbose
+RUN strip target/x86_64-alpine-linux-musl/release/openethereum
 
 FROM alpine:edge
 
 # show backtraces
 ENV RUST_BACKTRACE 1
 
+# curl and jq are installed to help create health and readiness checks on Kubernetes
 RUN apk add --no-cache \
   libstdc++ \
   eudev-libs \
-  libgcc
+  libgcc \
+  curl \
```

Comment:
Is this a backport of other issue?

Reply:
I tried clang and it fails to compile kvdb this is why I left gcc. 

For curl it is used by Kubernetes or at least it says that in newest docker file: `# curl and jq are installed to help create health and readiness checks on Kubernetes`